src/components/member-profile/index.js

```diff
@@ -367,6 +374,11 @@ const Profile = (props) => {
           {children}
         </Modal>
       )}
+      {isSuperUser && (
+        <div id="memberRoleUpdateModal">
+          {showMemberRoleUpdateModal && <MemberRoleUpdate />}
+        </div>
+      )}
```

Comment:
shouldn't this be like this:

```js
{isSuperUser && showMemberRoleUpdateModal  && (
    <div id="memberRoleUpdateModal">
      <MemberRoleUpdate />
    </div>
)}
```

Reply:
what you are mentioning is correct but can not be done here because `react.createPortal` is not allowing to do so,